source/custom/nifake_non_ivi_service.custom.cpp

```diff
@@ -1,8 +1,94 @@
 #include <nifake_non_ivi/nifake_non_ivi_service.h>
+#include <server/async_operation.h>
+#include <server/async_processing_loop.h>
 #include <server/callback_router.h>
 
+#include <memory>
+#include <thread>
 namespace nifake_non_ivi_grpc {
 
+class ReadStreamOperation : public nidevice_grpc::CompletionQueueElement {
```

Comment:
Note: When I add the other 2 DAQ Register callback methods, I will implement codegen for these "operation" classses and `register_async_functions` but *not* `process_XYZ`.

So this block will be pulled out.